include/h2o.h

```diff
@@ -1085,6 +1087,13 @@ struct st_h2o_req_t {
      */
     H2O_VECTOR(h2o_req_error_log_t) error_logs;
 
+    /**
+     * handler specific log data
+     */
+    struct {
+        h2o_proxy_log_data_t *proxy;
+    } handler_log_data;
```

Comment:
Two questions here:
* does `proxy` need to be a pointer?
* do we need to wrap `h2o_proxy_log_data_t` with a struct named `handler_log_data`? I'd assume that the timing information we will be logging directly in h2o to be i) when the information is sent to the client and ii) when the information is generated by the generator. If my assumption is true, then we will have nothing other than `proxy`.